lib/kernel-manager.js

```diff
@@ -86,11 +86,18 @@ class KernelManager {
 
   startKernel(kernelSpec, grammar, onStarted) {
     const displayName = kernelSpec.display_name;
+    let kernelStartDir;
 
     log("KernelManager: startKernel:", displayName);
 
+    if (atom.config.get("Hydrogen.startInProjectRootDir")) {
+      kernelStartDir = atom.project.getPaths()[0];
```

Comment:
I think `atom.project.getPaths()` can be empty if no project is present in the tree view. In that case we should fall back to the `getEditorDirectory(store.editor)`.

A very nice touch (now or in a future PR) would be to use the project where the current file belongs to and not just the first one. This could be handy if multiple projects are open.